src/modules/shlvl.rs

```diff
@@ -22,11 +22,19 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 
     let shlvl_str = &shlvl.to_string();
 
-    let repeat_count = if config.repeat {
+    let mut repeat_count: usize = if config.repeat {
```

Comment:
`repeat_count` was implicitly `usize`, I'm sticking to that.

Reply:
Used `u64` for config param `repeat_offset`.

`repeat_count` is used in repeat calls which need `usize` so nothing to gain from changing it's type.